types/make.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
```

Comment:
Haven't tried, but I don't think it will work. The reason you can use binaries in dependencies is that when npm install binaries it creates a "shim" on Windows that makes it behave like Unix.